Libraries/Image/RCTImageLoader.m

```diff
@@ -480,48 +457,25 @@ - (RCTImageLoaderCancellationBlock)loadImageWithURLRequest:(NSURLRequest *)image
   __block void(^cancelLoad)(void) = nil;
   __weak RCTImageLoader *weakSelf = self;
 
-  // Check decoded image cache
-  NSString *cacheKey = RCTCacheKeyForImage(imageURLRequest.URL.absoluteString, size, scale, resizeMode);
-  {
-    UIImage *image = [_decodedImageCache objectForKey:cacheKey];
-    if (image) {
-      // Most loaders do not return on the main thread, so caller is probably not
-      // expecting it, and may do expensive post-processing in the callback
-      dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0), ^{
```

Comment:
We used to dispatch  the completion block on a background thread, now you're calling it immediately.